server/router.js

```diff
@@ -13,4 +14,5 @@ router.post('/login', login);
 router.get('/logout', logout);
 router.post('/event', auth, postEvent);
 router.get('/event/:id', auth, getEvent);
+router.put('/event/:id', auth, updateEvent);
```

Comment:
Rename the name of the function in a way better than this, ```approveEvent``` is clearer than ```updateEvent```

Reply:
done